Workflow/Sources/Workflow.swift

```diff
@@ -83,22 +83,51 @@ public protocol Workflow: AnyWorkflowConvertible {
     func render(state: State, context: RenderContext<Self>) -> Rendering
 }
 
-extension Workflow {
-    public func workflowDidChange(from previousWorkflow: Self, state: inout State) {}
+public extension Workflow {
```

Comment:
The formatter seems to have changed some things; I didn't do this one myself.

Reply:
Yeah I think at some point the default for the formatter changed or something? I dunno, but I want to bring the previous style back.

Reply:
I guess it picked up a newer version of `SwiftFormat`. I should probably pin this to a specific version.